### PR TITLE
fix: Fix code mirror long content load issue and font []

### PR DESCRIPTION
**Pull request by Astitva877**
## Description
Fixed the issue of content load when response is too long and also changed the font size and family 

image

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/lib/components/editor/codeMirrorTheme.ts [hunk 1]
```diff
@@ -48,11 +48,14 @@ import {
...
 } from "codemirror-json-schema";
 import { hoverTooltip } from "@codemirror/view";
+
 export const basicTheme = EditorView.theme({
   "&": {
     height: "100%",
     width: "100%",
     flex: "1",
+    fontSize: "80%",
+    fontFamily: "'Source Code Pro', monospace",
```
Comment (LordNayan) [inline src/lib/components/editor/codeMirrorTheme.ts:58]: Can we take it from a constant

Comment (Astitva877) [reply src/lib/components/editor/codeMirrorTheme.ts]: Resolved

**Review by LordNayan — APPROVED** [review]

**Pull request merged**